src/main/java/org/dcsa/core/validator/VesselIMONumberValidator.java

```diff
@@ -1,10 +1,7 @@
 package org.dcsa.core.validator;
 
-import org.dcsa.core.util.ValidationUtils;
-
 import javax.validation.ConstraintValidator;
 import javax.validation.ConstraintValidatorContext;
-import java.util.regex.Pattern;
 
 public class VesselIMONumberValidator implements ConstraintValidator<ValidVesselIMONumber, String> {
```

Comment:
should we add final to this? Or do we expect variations of VesselIMONumberValidators through extending it?

Reply:
Done